google/api/control/check_request.py

```diff
@@ -238,6 +239,13 @@ def as_check_request(self, timer=datetime.utcnow):
         if self.referer:
             labels[_KNOWN_LABELS.SCC_REFERER.label_name] = self.referer
 
+        # Forcibly add system label reporting here, as the base service
+        # config does not specify it as a label.
+        labels['servicecontrol.googleapis.com/service_agent'] = (
```

Comment:
Doesn't this fit on one line? I'd also change these to use _KNOWN_LABELS.SCC_SERVICE_AGENT.label_name and so on.